src/test/groovy/io/sdkman/service/TextServiceSpec.groovy

```diff
@@ -27,6 +28,13 @@ class TextServiceSpec extends Specification {
     Broadcast broadcast1 = new Broadcast(id: 1, text: "text1", date: new Date())
     Broadcast broadcast2 = new Broadcast(id: 2, text: "text2", date: new Date())
 
+
+    Date iso8601Date = new Date().copyWith(
+            year: 2019,
+            month: Calendar.FEBRUARY,
+            dayOfMonth: 11)
+    Broadcast iso8601Broadcast = new Broadcast(id: "3",text: "Groovy 2.7.1 released!", date: iso8601Date)
+
```

Comment:
I would move this into the test body itself so everything is close together. Spock allows for a `given:` block where this code would fit well.

Reply:
Ok, will change. That what I've done initially, but I moved it up to keep it consistent with the other two broadcasts.